content/apps/using-oauth2.md

```diff
@@ -0,0 +1,157 @@
+# Using OAuth 2.0 to connect an App
+
+The PIM uses OAuth 2.0 to manage the authorization of Apps.
+
+![App activation diagram](../img/apps/app-activation-sequence-diagram.png)
```

Comment:
On the diagram, you would remove the bubble with 'User consent blabla' and be more explicit on the 4. user action by changing the text ''Confirm scope authorization" or something like that.
As this diagram destination is the developer's community, I would enhance it with more technical details like the requests and responses details. What do you think?

Reply:
I think it should not include the requests and responses, those are already documented, step by step, in the same page.